### DISCUSSION TITLE
alist挂迅雷就算在手机号前面加+86现在也不能用了，不知道什么问题，有谁知道呢

### DISCUSSION THREAD

**Discussion by kaixinxiaowangzi666** [AlistGo/alist#6223] [General]
![Snipaste_2024-03-15_15-40-03](https://github.com/alist-org/alist/assets/144104602/65189788-8ccd-40f0-85f4-3bf2addbb868)

**Comment by XrayjakerZara123** [comment]
兄弟，你和我一样情况，就是今天变成这样了

**Reply by kaixinxiaowangzi666** [reply]
就是

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5

**Comment by feicun2** [comment]
我的迅雷昨天同样问题

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5

**Comment by pongfcnkl** [comment]
直接抓包，这玩意复制到验证码里
![截图_20240318182531](https://github.com/alist-org/alist/assets/24273319/06b6889c-4f38-41b2-84d4-7110820a8ef1)

**Reply by kaixinxiaowangzi666** [reply]
用抓包工具抓的token复制到验证码框还是不行，

**Reply by pongfcnkl** [reply]
@kaixinxiaowangzi666 
你是怎么抓的？

**Reply by kaixinxiaowangzi666** [reply]
你抓的token填写后传奇alist就不会出问题吗？

**Reply by pongfcnkl** [reply]
@kaixinxiaowangzi666 
没有问题 一切正常

**Reply by kaixinxiaowangzi666** [reply]
你可以详细说一下怎么弄好的

**Reply by pongfcnkl** [reply]
根据我截图直接抓包填入验证码就行了啊

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5

**Comment by wow8szh** [comment]
+86后面加个空格再输入电话号码就可以了

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5
